### PR TITLE
(gimp) Embedded package + added innosetup logging

### PR DESCRIPTION
**Pull request by AdmiringWorm**

### CODE REVIEW

File: automatic/gimp/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -1,39 +1,15 @@
...
-  checksumType   = 'sha256'
-  checksumType64 = 'sha256'
-
-  # The installer contains both 32-bit and 64-bit versions of GIMP, and will automatically use the appropriate one.
-  # Text quoted from <https://www.gimp.org/downloads/>, look below download buttons.
-  url            = 'https://download.gimp.org/mirror/pub/gimp/v2.8/windows/gimp-2.8.22-setup.exe'
-  url64bit       = 'https://download.gimp.org/mirror/pub/gimp/v2.8/windows/gimp-2.8.22-setup.exe'
-
-  silentArgs     = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-'
+  silentArgs     = "/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP- /LOG=`"$($env:TEMP)\$($packageName).$($env:chocolateyPackageVersion).InnoSetup.log`""
```
Comment (majkinetor) [inline automatic/gimp/tools/chocolateyInstall.ps1]:
```
 /LOG=`"$( Join-Path (Get-PackageCacheLocation) InnoSetup.log )"`
```

File: automatic/gimp/tools/chocolateyInstall.ps1 [hunk 2]
```diff
@@ -1,39 +1,15 @@
...
-if ($installedVersion -eq $version) {
-  Write-Output $(
-    "GIMP $installedVersion is already installed. " +
-    "Skipping download and installation."
-  )
-} else {
-  Install-ChocolateyPackage @packageArgs
-}
+Install-ChocolateyInstallPackage @packageArgs
+Remove-Item -Force -ea 0 "$toolsPath\*.exe","$toolsPath\*.ignore"
```
Comment (majkinetor) [inline automatic/gimp/tools/chocolateyInstall.ps1]: This needs that line with `touch` if rm failed to delete for some reason.

File: automatic/gimp/tools/chocolateyInstall.ps1 (hunk 1 above)
Comment (AdmiringWorm) [reply automatic/gimp/tools/chocolateyInstall.ps1]: The idea is to log to a file in the same location as the ones used when running an msi installer, without creating uneeded directories. (I noticed I had included the chocolatey-core.extension though, which I need to remove)

File: automatic/gimp/tools/chocolateyInstall.ps1 (hunk 2 above)
Comment (AdmiringWorm) [reply automatic/gimp/tools/chocolateyInstall.ps1]: I'll update, but can't use `touch` as that isn't available in POSHv2

**Pull request merged**